.flake8

```diff
@@ -0,0 +1,10 @@
+[flake8]
+# E111 indentation is not a multiple of four
+# E114 indentation is not a multiple of four (comment)
+# E121 continuation line under-indented for hanging indent
+# E123 closing bracket does not match indentation of opening bracket's line
+# E129 visually indented line with same indent as next logical line
+# E402 module level import not at top of file
+# W503 line break before binary operator
+# W605 invalid escape sequence '\d'
```

Comment:
It seems regular expression literals without raw ("r") prefix cause this. It also causes DeprecationWaring since Python 3.6 and it will be SyntaxError [1].  So I think it is next step that we'll remove W605 from ignore list.

[1] https://docs.python.org/3/whatsnew/3.6.html#deprecated-python-behavior

Reply:
So, remove W605 from the exceptions list and make sure all our regexp's have the `r` prefix?  Got it.

Reply:
I decided against trying to do a blanket conversion to `r` strings for every regular expression.  But I think I fixed the affected ones.

Reply:
I confirmed. Thank you.  I don't think that it is need for all regular expressions. It only needs for the affected ones.